spec/models/votation_type_spec.rb

```diff
@@ -27,9 +27,52 @@
 
     expect(votation_type).to be_valid
 
+    votation_type.vote_type = "open"
+
+    expect(votation_type).to be_valid
+
     votation_type.vote_type = "multiple"
 
     expect(votation_type).not_to be_valid
     expect(votation_type.errors[:max_votes]).to include "can't be blank"
   end
+
+  describe "#cannot_be_open_ended_if_question_has_options" do
+    it "allows changing to open-ended when the question has no options" do
+      poll_question = create(:poll_question_unique)
+      votation_type = poll_question.votation_type
+
+      expect(votation_type.update(vote_type: :open)).to be true
+      expect(votation_type).to be_valid
+      expect(votation_type.errors[:vote_type]).to be_empty
+    end
+
+    it "blocks changing to open-ended when the question has options" do
+      poll_question = create(:poll_question_unique)
+      create(:poll_question_option, question: poll_question)
+      votation_type = poll_question.votation_type
```

Comment:
Since we're testing votation types, creating votation types might feel more natural :thinking: . What do you think about:

```suggestion
      votation_type = create(:votation_type_unique)
      create(:poll_question_option, question: votation_type.questionable)
```